test/utils.ts

```diff
@@ -44,29 +46,29 @@ export function createApiCall(func, opts?) {
   opts = opts || {};
   const settings = new gax.CallSettings(opts.settings || {});
   const descriptor = opts.descriptor;
-  return apiCallable.createApiCall(
-      Promise.resolve((argument, metadata, options, callback) => {
-        if (opts.returnCancelFunc) {
-          return {
-            cancel: func(argument, metadata, options, callback),
-            completed: true,
-            call: () => {
-              throw new Error('should not be run');
-            }
-          };
-        }
-        func(argument, metadata, options, callback);
-        return {
-          cancel: opts.cancel || (() => {
-                    callback(new Error('canceled'));
-                  }),
-          completed: true,
-          call: () => {
-            throw new Error('should not be run');
-          }
-        };
-      }),
-      settings, descriptor);
+  return realCreateApiCall(
+             Promise.resolve((argument, metadata, options, callback) => {
+               if (opts.returnCancelFunc) {
+                 return {
+                   cancel: func(argument, metadata, options, callback),
+                   completed: true,
+                   call: () => {
+                     throw new Error('should not be run');
+                   }
+                 };
+               }
+               func(argument, metadata, options, callback);
+               return {
+                 cancel: opts.cancel || (() => {
+                           callback(new Error('canceled'));
+                         }),
+                 completed: true,
+                 call: () => {
+                   throw new Error('should not be run');
+                 }
+               };
+             }),
+             settings, descriptor) as GaxCallPromise;
```

Comment:
This is mostly caused by typings. Since we now state we return `GaxCallPromise` and it states it accepts `(argument: {}, callOptions?: CallOptions, callback?: APICallback)`, `null` does not really fit into `callOptions`. I could've made `callOptions?: CallOptions|null` but it's just ugly and not really needed.